apps/account-api/src/main.ts

```diff
@@ -37,10 +39,11 @@ async function bootstrap() {
   });
 
   const app = await NestFactory.create<NestExpressApplication>(ApiModule, {
-    logger: getLogLevels(),
     rawBody: true,
   });
 
+  app.useLogger(app.get(PinoLogger));
```

Comment:
This `useLogger` call is only ever used when constructing an app, so only in `main.ts` files.